client/actions/documentActions.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * Documents action, disptach action and
+ * action types of each action to the reducer
+ */
+
+import axios from 'axios';
+import types from './actionTypes';
+
+/**
+ *
+ * loaddocumentsuccess
+ * @export
+ * @param {any} documents  returned documents from api call
```

Comment:
I also feel the params isn't just anything ?. I'm guessing it's an **Array** of documents ?